firebase-functions/functions/src/firebase/calculateResult.ts

```diff
@@ -14,7 +14,16 @@ export const calculateResult = functions.firestore.document('/players/{documentI
     try {
       const roomId = await getCurrentRoomId(context.params.documentId as string);
       const initiallyActivePlayers = await getActivePlayersInRoom(roomId);
-      if (!hasEveryoneChosen(initiallyActivePlayers)) {
+      const numberOfPlayersInTheRoom = await getNumberOfPlayers(roomId);
+      const isThereOnlyOnePlayerInTheRoom = numberOfPlayersInTheRoom < 2;
```

Comment:
This method is getting a little long, maybe some things can be extracted? I think I see some opportunities to save on some lines :)

Reply:
Pretty please, we have a story for refactoring, I promise I will have a look at it later :)
Here a story for that
https://github.com/users/DanielMuckelbauer/projects/1?pane=issue&itemId=19073454

Reply:
Alright, I updated that task a little, although your description was of course absolutely precise and complete already! Let's just do it early please. It's rotting :(